sklearn/neighbors/_binary_tree.pxi

```diff
@@ -804,17 +789,11 @@ cdef class BinaryTree:
     # Use cinit to initialize all arrays to empty: this will prevent memory
     # errors and seg-faults in rare cases where __init__ is not called
```

Comment:
The suggestion for comments that I added previously can complete or be completed by this comment.